lib/kvdb/kvdb_rparams.c

```diff
@@ -391,6 +391,145 @@ mclass_policies_validator(const struct param_spec *ps, const void *data)
     return true;
 }
 
+static merr_t
+mclass_array_build(cJSON *const arr, const uint8_t *const dtype)
+{
+    cJSON *mclass = NULL;
+
+    INVARIANT(arr);
+    INVARIANT(cJSON_IsArray(arr));
+    INVARIANT(dtype);
+
+    for (int i = 0; i < MP_MED_COUNT; i++) {
+        if (dtype[i] == MP_MED_INVALID)
+            continue;
+
+        mclass = cJSON_CreateString(mpool_mclass_to_string[i]);
+
+        if (!mclass)
+            return merr(ENOMEM);
+
+        cJSON_AddItemToArray(arr, mclass);
+    }
+
+    return 0;
+}
+
+static merr_t
+mclass_policies_stringify(
```

Comment:
Seems like this will generate a huge conflict with my PR? I've removed the mclass policy allocation fallback, so the target media class for a dtype (keys/values) is now specified as a JSON string instead of a JSON array.

```
{
    "name": "capacity_only",
    "config": {
        "root": {
            "keys": "capacity",
            "values": "capacity"
        },
        .....
    }
}
```

Reply:
It would cause a little bit of conflict, but you can just replace it with you implementation.